vivado/v2020_1/Makefile.sysc.rules

```diff
@@ -0,0 +1,502 @@
+#===- Makefile.sysc.rules ----------------------------------*- Makefile -*--===#
```

Comment:
remove this file. It is not included in the glob

Reply:
guess you still got all the other ones there too. maybe it is better to leave unmodified